### PR TITLE
Counter

### PR DESCRIPTION
**Pull request by darionara**

### CODE REVIEW

File: css/counter.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+:root {
+    --light-blue: rgb(223, 235, 250);
```
Comment (tomekskuta) [inline css/counter.css:2]: CSS variables 💪

File: css/counter.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    --dark-blue: rgb(16, 74, 175)
+}
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+main {
```
Comment (tomekskuta) [inline css/counter.css]: it looks rather like `main.css` stuff than `counter.css` 😄 I mean styling `main` element is global thing not exactly related to the counter itself - positioning some component inside the view is a job for the view and not for component :)

File: css/counter.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    height: 4rem;
+    border: 2px solid var(--medium-blue);
+    border-radius: 1rem;
+    font-size: 2rem;
+    font-weight: 700;
+    color: var(--dark-blue);
+    text-align: center;
+}
+
+.counter__input:focus-visible {
```
Comment (tomekskuta) [inline css/counter.css:33]: `:focus-visible` 😎 not obvious choice but the correct one 😄

File: index.html [hunk 4]
```diff
@@ -6,7 +6,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>First Project</title>
     <link rel="stylesheet" href="./css/main.css">
+    <link rel="stylesheet" href="./css/counter.css"></link>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
```
Comment (tomekskuta) [inline index.html:11]: didn't know these google icons and how simple is their usage 👌 thanks :)

File: index.html [hunk 5]
```diff
@@ -35,10 +37,22 @@
   </header>
 
     <main>
-
+      <div class="counter">
+        <button class="counter__sign" id="counter__sign--minus">
```
Comment (tomekskuta) [inline index.html]: no need to use BEM convention in `ids` :) `camelCase`, `kebab-case` or `snake_case` are most popular and good practices here 👍

File: js/counter.js
Comment (tomekskuta) [inline js/counter.js:1]: really cool 👌 simple and readable 😎

**Review by tomekskuta — CHANGES_REQUESTED** [review]
Really good Counter 👍 no overengineering, simple and readable 💪 only two simple things in comments to change :)

File: css/counter.css (hunk 3 above)
Comment (darionara) [reply css/counter.css:33]: Thanks, I've spent some time to discover it! 😄 That is the last example of the element's states in DevTools xd

File: index.html (hunk 5 above)
Comment (darionara) [reply index.html]: I didn't know all of these names but I think kebab suits me 👌

**Review by tomekskuta — APPROVED** [review]
👌

**Pull request merged**